mithril-aggregator/config/preview.json

```diff
@@ -0,0 +1,16 @@
+{
+    "cardano_cli_path": "cardano-cli",
+    "cardano_cli_socket_path": "/tmp/cardano.sock",
+    "network": "preview",
+    "network_magic": 2,
```

Comment:
suggestion: use one or the other but not both of symbolic name for `preview` and `network_magic`. If there's an internal mapping from symbolic name to magic, you might want to use that.

Reply:
The known networks (`preview`, `preprod` and `testnet`) are now automatically mapped to their `network_magic`